From cfef6df70f9afde3764ecdeeb020b817114e486f Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Wed, 20 Dec 2023 16:06:33 +0800
Subject: [PATCH 17/77] clk: rockchip: rk3036: make armclk as critical

We found armclk maybe closed in some case,
for example, armclk and mac_clk_ref both under
apll, mac_clk_ref may close apll when it probe
finished.

Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 drivers/clk/rockchip/clk-rk3036.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/rockchip/clk-rk3036.c b/drivers/clk/rockchip/clk-rk3036.c
index d789a09b35d08..6a9c8705e9ee2 100644
--- a/drivers/clk/rockchip/clk-rk3036.c
+++ b/drivers/clk/rockchip/clk-rk3036.c
@@ -429,6 +429,7 @@ static struct rockchip_clk_branch rk3036_clk_branches[] __initdata = {
 static const char *const rk3036_critical_clocks[] __initconst = {
 	"aclk_cpu",
 	"aclk_peri",
+	"armclk",
 	"hclk_peri",
 	"pclk_peri",
 	"pclk_ddrupctl",
-- 
2.47.0

